.github/actions/frontend/runtime/e2e_prepare/README.md

```diff
@@ -0,0 +1,36 @@
+# Frontend Runtime - E2E Prepare
+
+This [composite action](./action.yml) is responsible for executing [Artemis](https://github.com/JupiterOne/artemis) and obtaining the necessary information needed for logging in via the account/user created.
+
+## Inputs
+
+| Name                        | Type    | Default                      | Required  | Description                                               |
+| --------------------------- | ------- | ---------------------------- | --------- | --------------------------------------------------------- |
+| `e2e_artemis_config_path`   | String  | cypress/artemis-config.yaml  | False     | Used to determine the path to the artemis config file
+| `userCount`                 | String  |                              | True      | The number of tests that you want Cypress to run in parallel (obtained via `$(echo '${{ inputs.e2e_containers }}' | jq '. | length')`)
```

Comment:
This is listed as require here, but has a default value in the action code.

Reply:
Good call, this should be false. Updated.